modules/renter/hostdb/scan.go

```diff
@@ -242,40 +250,52 @@ func (hdb *HostDB) managedScanHost(entry modules.HostDBEntry) {
 	hdb.mu.Unlock()
 }
 
+// runs scan on a host
+func (hdb *HostDB) threadedProbeHost(hostEntry modules.HostDBEntry) {
+	// Block the scan until the host is online.
+	for {
+		hdb.mu.RLock()
+		online := hdb.online
+		hdb.mu.RUnlock()
+		if online {
+			break
+		}
+	}
```

Comment:
This is a very tight loop that will lock up the CPU. Need to add in the 30 second soft sleeps that were in the previous implementation​ of this loop.